### PR TITLE
Oxedions/fix/website

### PR DESCRIPTION
**Pull request by oxedions**
Only second commit is interesting. The first one is only a sync with current website. I will try to keep it sync from now.

### CODE REVIEW

File: website/the_stack/features.html [hunk 1]
```diff
@@ -139,7 +139,7 @@ <h2 class="title">
 
 <div class="columns is-vcentered">
   <div class="column">
-<span style="color: #108bde">Blue</span>Banquise design was made to ensure full modularity.
+<span style="color: #108bde">Blue</span>Banquise can be seen as an Ansible collection of roles, which ensure full modularity.
```
Comment (btravouillon) [inline website/the_stack/features.html:142]: I would rephrase: ...as a set of Ansible roles, which ... 

This would be misleading to talk of Ansible collection on the website, because [Ansible Collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) exist and BlueBanquise is not compatible (yet?).

**Review by btravouillon — CHANGES_REQUESTED** [review]

File: website/the_stack/features.html (hunk 1 above)
Comment (oxedions) [reply website/the_stack/features.html]: I agree. Corrected.

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

Note: I did not review first commit which might be the HTML rendering of the sources in the main project.

**Comment by oxedions** [comment]
Yes, first commit is a plain copy of the current website online.

Thanks a lot for the review !
